### PR TITLE
common: make Append() logic more easier to read

### PR DESCRIPTION
**Pull request by asddongmen**
### Check List 

Tests 

 - No code

Code changes

Side effects

Related changes

### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/sink/common/common.go [hunk 1]
```diff
@@ -80,22 +80,29 @@ func (c *UnresolvedTxnCache) Append(filter *filter.Filter, rows ...*model.RowCha
...
-			c.unresolvedTxns[row.Table.TableID] = txns
+			}
+			// it will be grouped by startTs in Append()
+			txns.Append(row)
+			txnsGroup = append(txnsGroup, txns)
+			c.unresolvedTxns[row.Table.TableID] = txnsGroup
+		} else if txnsGroup[lastIndex].commitTs == row.CommitTs {
+			txnsGroup[lastIndex].Append(row)
+		} else {
+			log.Panic("the commitTs of the emit row is less than the received row",
```
Comment (overvenus) [inline cdc/sink/common/common.go:100]: I prefer to state panic conditions explicitly.

```go
if invariant not hold {
    panic()
}
```

Comment (asddongmen) [reply cdc/sink/common/common.go:100]: got it

**Pull request closed without merging**